component/site/libraries/helper.php

```diff
@@ -1670,67 +1671,67 @@ function canPublishOwnEvents($evid, $vevent = false)
 			{
 				return true;
 			}
-                        
```

Comment:
Darn I wish Git would allow better understanding of code-refactor. 

I've just added: $canPublishOwn = false; to the top of the function and then if $canpublishown = true and no other false conditions are met at the end it returns true to canpublish own. 

Then in JEvpublished, it also shows if they can publishownevents.